make/autoconf/spec.gmk.in

```diff
@@ -889,6 +889,9 @@ USE_EXTERNAL_LIBPNG:=@USE_EXTERNAL_LIBPNG@
 PNG_LIBS:=@PNG_LIBS@
 PNG_CFLAGS:=@PNG_CFLAGS@
 
+ENABLE_LIBSLEEF:=@ENABLE_LIBSLEEF@
```

Comment:
You need to merge in the latest changes from mainline. I have just updated spec.gmk.in to always have a space around `:=`. Please do so with your additions as well.